### ISSUE TITLE
okular warns about missing breeze icon theme

### ISSUE THREAD

**Issue by cdepillabout** [NixOS/nixpkgs#53750]
## Issue description

When starting `okular`, it gives warnings that an icon theme is not installed:

```sh
$ okular /path/to/my.pdf
Icon theme "breeze" not found.
Icon theme "breeze" not found.
Error: standard icon theme "breeze" not found!
Icon theme "breeze" not found.
Icon theme "breeze" not found.
Error: standard icon theme "breeze" not found!
```

What's going on here?  Do I have to manually install the breeze icon theme?  Would this be the `breeze-icons` package?

As an end-user, it would be nice if the `okular` derivation just pulled in an icon theme that would be used if I don't already have one installed.  If it is known that `okular` gives warnings without an icon theme, it would be nice if it just worked correctly when installed.

### Steps to reproduce

I've installed `okular` with `nix-env` in 18.09.

In my `/etc/nixos/configuration.nix`, I've enabled the X server, but I don't have any desktop environment enabled:

```nix
{
   services.xserver.enable = true;
}
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.90, NixOS, 18.09.1822.0a8f35c998f (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(root): `"nixos-18.09.1822.0a8f35c998f"`
 - channels(illabout): `"unstable-19.03pre164715.201d739b0ff"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by cdepillabout** [comment]
Pinging @ttuegel and @vandenoever, since it looks like they are maintainers for the KDE applications.

**Comment by FRidh** [comment]
Fixed in https://github.com/NixOS/nixpkgs/pull/54525

**Comment by newptcai** [comment]
I have the same problem with version 21.12.3

**Comment by opennota** [comment]
@newptcai Same here. `apt install breeze-icon-theme` has fixed that.

**Comment by me-at-gitt** [comment]
Same here.  `apt install breeze-icon-theme` has fixed that for me for okular 21.12.3. But this is just a fix, not a solution.

**Comment by JohnAZoidberg** [comment]
> Same here. `apt install breeze-icon-theme` has fixed that for me for okular 21.12.3. But this is just a fix, not a solution.

What? Are you using nix on Debian/Ubuntu? If you're not using nix, then this is the wrong repo to comment on.
I don't see this warning on GNOME/Wayland on NixOS 24.05
